packages/frontend-shared/src/components/Select.vue

```diff
@@ -177,6 +183,12 @@ const props = withDefaults(defineProps<{
   itemKey: 'key',
 })
 
+const isSelectedOption = (option: Option) => {
+  const optionKey = get(option, props.itemKey)
+
+  return optionKey && optionKey === get(props.modelValue, props.itemKey)
+}
+
```

Comment:
@mschile Thanks for showing us the issue. we found out that we are not keeping ref between `modelValue` and `option` in usages, to resolve it we use `itemKey` to check if an option is selected. WDYT about this way?